demos/benchmarking/docker-jans-loadtesting-jmeter/Dockerfile

```
@@ -61,7 +61,7 @@ ENV FQDN="https://demoexample.jans.io" \
 # ======
 
 COPY requirements.txt /scripts/requirements.txt
-RUN pip install --no-cache-dir -U pip wheel \
+RUN pip install --no-cache-dir -U --force-reinstall pip wheel \
```

Hadolint (2.15.1)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)